components/clp-py-utils/clp_py_utils/clp_metadata_db_utils.py

```
@@ -14,6 +14,38 @@
 )
 
 
+def _generic_get_table_name(prefix: str, suffix: str, dataset: str | None) -> str:
+    table_name = prefix
+    if dataset is not None:
+        table_name += f"{dataset}_"
+    table_name += suffix
+    return table_name
+
+
+def get_archives_table_name(table_prefix: str, dataset: str | None) -> str:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)